src/Integration/VsSessionHost.cs

```diff
@@ -51,7 +52,7 @@ internal sealed class VsSessionHost : IHost, IProgressStepRunnerWrapper, IDispos
 
         [ImportingConstructor]
         public VsSessionHost([Import(typeof(SVsServiceProvider))] IServiceProvider serviceProvider, SonarQubeServiceWrapper sonarQubeService, IActiveSolutionTracker solutionTacker)
-            : this(serviceProvider, null, null, sonarQubeService, solutionTacker, new SolutionBinding(serviceProvider), Dispatcher.CurrentDispatcher)
```

Comment:
Don't need SolutionBinding to be injected at this point, fetching it from a service just like in other places.